### PR TITLE
feat: array field element duplicate functionality

### PR DESCRIPTION
**Pull request by siarhei-zharnasek**
First of all, thanks for such awesome library.

For some purposes it's useful to have duplicate functionality not only for the whole component but also for Array field element itself.

image

### CODE REVIEW

**Review by chrisvxd — CHANGES_REQUESTED** [review]
Great feature! Thank you for adding it. I left some comments.

Once you've address, I'll hold off merging until we're closer to v0.17.

File: packages/core/components/AutoField/fields/ArrayField/index.tsx
Comment (siarhei-zharnasek) [reply packages/core/components/AutoField/fields/ArrayField/index.tsx:250]: sure, my bad
changed order

File: packages/core/components/AutoField/fields/ArrayField/index.tsx
Comment (siarhei-zharnasek) [reply packages/core/components/AutoField/fields/ArrayField/index.tsx]: okay, done

**Review by chrisvxd — APPROVED** [review]
🎉

**Pull request merged**
